TabloidMVC/Controllers/AccountController.cs

```diff
@@ -50,6 +50,7 @@ public async Task<IActionResult> Login(Credentials credentials)
             return RedirectToAction("Index", "Home");
         }
 
+        // Logout will redirect to home page, where user will have option to log back in
```

Comment:
Works as expected! Great job. In general, we probably don't need a full PR for a comment-- typically we'd just add comments as we go, or if we notice a lot of functionality is missing documentation we could add comments for multiple features at once and try to group them all on the same PR. Just a note for next time :)